### PR TITLE
Add basic unit tests for REST API and rename app.py to plino_app.py

### PR DESCRIPTION
**Pull request by sahildua2305**
A few points -

- I have renamed the app.py to `plino_app.py` to make the names a bit better from testing point of view. Because once I started writing `test_app.py` for `app.py`, I realized that I had to use `app.app.post()` wherever now I'm using `plino_app.app.post()` that makes more sense.
- I have made corresponding changes in makefile as well.

Please have a look and let me know if I missed some link after renaming the `app.py` file.

### CODE REVIEW

**Comment by prodicus** [comment]
Hey @sahildua2305 

Some suggestions if you may

- Can you put the file `plino/test_plino_app.py` inside a project folder like `tests//test_plino_app.py`. This looks cleaner and makes more sense. The name of the test file could depict what it is testing for too.

```sh
.
├── CONTRIBUTING.md
├── tests
│   └── test_plino_app.py
├── LICENSE
├── logfiles
│   └── logfile.txt
├── Makefile
├── plino
│   ├── app.py
│   ├── classifier.py
│   ├── __init__.py
│   ├── nltk_data
│   ├── saved_classifiers
│   ├── static
│   ├── templates
│   ├── train.py
│   └── utils.py
├── Procfile
├── README.md
└── requirements.txt
```

- Runnning the tests through the `make` command would save a lot of time.

What do you think?

**Comment by sahildua2305** [comment]
@prodicus Check now. Agreed with the changes you suggested, made corresponding changes.

File: tests/test_plino_app_api_response.py [hunk 1]
```diff
@@ -2,10 +2,9 @@
...
 # @Date:   2016-05-15 05:26:34
 # @Last Modified by:   Sahil Dua
-# @Last Modified time: 2016-05-15 06:24:11
+# @Last Modified time: 2016-05-16 00:00:36
 
 import os, json
-import plino_app
 import requests
 import unittest
 import tempfile
```
Comment (prodicus) [inline tests/test_plino_app_api_response.py]: Could be removed as it is no longer used.

**Comment by prodicus** [comment]
Works for me. :+1: 

Add `requests` in the `requirements.txt`

**Pull request merged**